hs/Database/Tables.hs

```diff
@@ -140,6 +141,13 @@ FacebookTest
 
 -- ** TODO: Remove these extra types and class instances
 
+-- | JSON SVG data
+data SvgJSON =
+    SvgJSON { texts :: [Text],
```

Comment:
It might be possible to delete it entirely. I only used it to use it with: `Just (SvgJSON texts shapes paths) -> do 
`